frontend/scripts/robot/components/detail.js

```diff
@@ -26,6 +28,8 @@ export default React.createClass({
     if (loading) {
       return <Loading/>;
     } else if (loadError) {
+      return <Error loadError={loadError}/>;
+    } else if (isEmpty(model)) {
       return <NotFound/>;
     } else {
```

Comment:
You see I change the logic:
so if we have errors, error component will show.
in model is not exist, NotFound component will show.

Related issue: https://github.com/Paqmind/react-starter/issues/36